src_wires_pub/wires.F90

```diff
@@ -5470,7 +5470,7 @@ subroutine AdvanceWiresE(sgg,timeinstant, layoutnumber,wiresflavor,simu_devia,st
           endif
       endif
       
-!!!230323 comento lo que sigue porque ya manejo fuentes duras y blandas antes, pero esto funcion� con jag rayos junio'15
+!!!230323 comento lo que sigue porque ya manejo fuentes duras y blandas antes, pero esto funcion� con jag rayos junio'15

```

Comment:
Seems that unicode characters are problematic for some editor. Please change to plain ASCII letters.